client/components/async-gridicons/index.jsx

```diff
@@ -9,14 +9,18 @@ import React from 'react';
  * Internal dependencies
  */
 import AsyncLoad from 'components/async-load';
+import FallbackIcon from './fallback';
 
 function AsyncGridicon( { icon = '', ...rest } ) {
 	return (
```

Comment:
This is marked red, not sure why though 🤔 

<img width="186" alt="screen shot 2018-09-24 at 18 01 22" src="https://user-images.githubusercontent.com/9202899/45963643-ded9bc80-c023-11e8-8673-f9f31d7ec785.png">